Fastfile

```diff
@@ -14,6 +14,15 @@ default_platform :ios
     scan(skip_slack: skip_slack)
   end
 
+  lane :ui_test do
+    _setup()
+    skip_slack = ENV['SCAN_SLACK_CHANNEL'].to_s.strip.empty?
+    scan(skip_slack: skip_slack,
+         devices: ["iPhone 8"],
+         output_types: "html",
```

Comment:
Again worth the output type being configurable, its fine to have a default but people should be able to choose